app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,64 @@
+class PasswordResetsController < ApplicationController
+  before_action :load_user, :valid_user, :check_expiration,
+                only: [:edit, :update]
+
+  def new; end
+
+  def create
+    @user = User.find_by email: params.dig(:password_reset, :email)&.downcase
+    if @user
+      @user.create_reset_digest
+      @user.send_password_reset_email
+      flash[:info] = t "flash_create_infor"
+      redirect_to static_pages_home_path
+    else
+      flash.now[:danger] = t "flash_create_danger"
+      render :new
+    end
+  end
+
+  def edit; end
+
+  def update
+    if user_params[:password].empty?
+      @user.errors.add(:password, t("error"))
+      render :edit
+    elsif @user.update(user_params)
+      log_in @user
+      @user.update_column(:reset_digest, nil)
+      flash[:success] = t "reset_password_success"
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  private
+
+  def load_user
+    @user = User.find_by(email: params[:email])
```

Comment:
bỏ () những chỗ ko cần thiết e nha
```suggestion
    @user = User.find_by email: params[:email]
```